src/online2/online-timing.cc

```diff
@@ -52,7 +52,7 @@ void OnlineTimingStats::Print(bool online){
     // real-time factor.
     double real_time_factor = (total_time_taken_ - total_time_waited_) /
         total_audio_;
-    KALDI_LOG << "Timing stats: real-time factor for offline decoding was "
+    KALDI_LOG << "Timing stats: real-time factor for online decoding was "
```

Comment:
I'd rather not change it.. that is actually telling you the real-time factor you'd get if it wasn't an online system, i.e. if you didn't have to wait for the input.